### ISSUE TITLE
The bullet in bullet text is not read aloud in the "Change PIN" screen.

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#2298]
### Description of problem
The bullet in bullet text is not read aloud in the "Change PIN" screen. The bullet gets highlighted but nothing is read aloud.

### Expected behavior
The bulleted text should be a bullet list where the bullet is read aloud.

### Environment
- [x] Occurs on Android
- [x] Occurs on iOS

Build #: 2109

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [x] Low

**Comment by cmathies16** [comment]
@knguyenBC is this still an issue?

**Comment by knguyenBC** [comment]
I won't be able to test if this is still an issue as I don't have access to the latest beta build of the BCSC app
